com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/publisher/Publisher.java

```diff
@@ -42,6 +43,8 @@ public String getManifestPrivateKey() {
         return this.settings.getManifestPrivateKey();
     }
 
+    public IResource getPublisherSettingsResorce() { return this.settings.getResource(); }
```

Comment:
We need resource file later to be able to use it as input for GameProject. Changes in LiveUpdate file should re-trigger GameProjectBuilder